src/libs/actions/connections/ConnectToXero.ts

```diff
@@ -0,0 +1,9 @@
+import {getCommandURL} from '@libs/ApiUtils';
+
+const getXeroSetupLink = (policyID: string) => {
+    const params = new URLSearchParams({policyID});
+    const commandURL = getCommandURL({command: 'ConnectPolicyToXero', shouldSkipWebProxy: true});
```

Comment:
is it the final version and we are sure we want to skip web proxy? 😄

Reply:
Right now proxy causes **407 Proxy Authentication Required** error. When we adjust backend to work with proxy then we can remove this, but let's not allow this to block us